### ISSUE TITLE
Strapi Build fails having this plugin for @strapi/strapi v.4.5.5 

### ISSUE THREAD

**Issue by woomiz** [boazpoolman/strapi-plugin-sitemap#104]
### Description
With the strapi version v.4.5.5 , after updating core strapi packages. Trying to build the strapi project dosn't work.

### Steps to reproduce the behavior

1. Upgrade all of the Strapi packages version numbers in package.json to the latest stable Strapi version:
```json
{
  "dependencies": {
    "@notum-cz/strapi-plugin-content-versioning": "^0.4.7",
    "@palmabit/strapi-app-version": "^1.1.0",
    "@strapi/plugin-graphql": "^4.5.5",
    "@strapi/plugin-i18n": "^4.5.5",
    "@strapi/plugin-seo": "^1.7.7",
    "@strapi/plugin-users-permissions": "^4.5.5",
    "@strapi/strapi": "^4.5.5",
    "better-sqlite3": "^8.0.1",
    "strapi-plugin-comments": "^2.2.1",
    "strapi-plugin-duplicate-button": "^1.0.4",
    "strapi-plugin-preview-button": "^1.0.1",
    "strapi-plugin-publisher": "^1.1.4",
    "strapi-plugin-sitemap": "^2.0.8",
    "strapi-plugin-slugify": "^2.3.2"
  }
}
```
2. Execute the build command `npm run build`

### Expected behavior

Plugin should be working fine after updating

### Output Error

> ModuleNotFoundError: Module not found: Error: Can't resolve '@fortawesome/react-fontawesome' in '/home/woozi/work/admire/2022/karthikaspices/website/backend-karthika/node_modules/strapi-plugin-sitemap/admin/src/components/PluginIcon'

### System

- Node.js version:  v14.21.2
- NPM version: 6.14.17
- Strapi version: "@strapi/strapi": "^4.5.5"
- Plugin version: "strapi-plugin-sitemap": "^2.0.8"
- Database: "better-sqlite3": "^8.0.1"
- Operating system: Ubuntu 22.04.1 LTS x86_64

**Comment by woomiz** [comment]
The Build works completely fine when I remove `strapi-plugin-sitemap` plugin

**Comment by boazpoolman** [comment]
Yep. I've put some description on why this is happening and how to fix it in #103.

**Comment by boazpoolman** [comment]
This is now fixed in the master branch. Will release probably tomorrow.

**Comment by boazpoolman** [comment]
This was fixed in [v2.0.9](https://github.com/boazpoolman/strapi-plugin-sitemap/releases/tag/2.0.9) released on NPM.
If you still experience this or another issue beyond v2.0.9 please feel free to report it here on Github.
